packages/components/rollup.config.mjs

```diff
@@ -27,13 +28,22 @@ const plugins = [
   // These are the modules that should get reexported into the traditional
   // "app" tree. Things in here should also be in publicEntrypoints above, but
   // not everything in publicEntrypoints necessarily needs to go here.
-  addon.appReexports([
-    'components/**/!(*types).js',
-    'helpers/**/*.js',
-    'modifiers/**/*.js',
-    'services/**/!(*types).js',
-    'instance-initializers/**/*.js',
-  ]),
+  addon.appReexports(
+    [
+      'components/**/!(*types).js',
+      'helpers/**/*.js',
+      'modifiers/**/*.js',
+      'services/**/!(*types).js',
+      'instance-initializers/**/*.js',
+    ],
+    {
+      mapFilename: (filename) => {
```

Comment:
This is the main change here, it makes sure that compatibility layer with ember-resolver and embroider mapping is correct